### ISSUE TITLE
Metamask+Trezor fail to sign EIP-712 message

### ISSUE THREAD

**Issue by drortirosh** [trezor/trezor-firmware#1543]
**Describe the bug**

Trying to use a dapp that requires EIP-712 (ctf-react.opengsn.org)
Works with Metamask account, but when using metamask with Trezor wallet we get an error message:
`MetaMask Message Signature: Error: Not supported on this device`

**Firmware version and revision**
latest firmware: 1.9.4
Describe firmware version and revision you are using when encountering the bug.

**Desktop/smartphone setup (please complete the following information):**
 - Wallet software: Metamask
 - OS: macOS
 - Browser chrome

**To Reproduce**
1. Go to ctf-react.opengsn.org
2. Connect your metamask to the dapp.
3. Click on "Capture the Flag"
4. see the "sign" message. click "Sign"
5. see the above error message.
6. (note that if you use with metamask "native" account, not connected to trezor, it does work as expected.)

**Expected behavior**
To sign properly

**Comment by prusnak** [comment]
No-one implemented EIP-712 signing on Trezor yet.

Duplicate of https://github.com/trezor/trezor-firmware/issues/1272

**Comment by prusnak** [comment]
EIP-712 for Trezor One is being developed here and will be hopefully released in January: https://github.com/trezor/trezor-firmware/pull/1970

**Comment by alisinabh** [comment]
@prusnak @matejcik Due to the lock in #1970 I cannot reply. Can you unblock me somehow? Many thanks.
(Sorry I wrote here)
